libs/localization/src/lib/en.json

```diff
@@ -1689,19 +1690,23 @@
         },
         "exactMatchRatio": {
           "name": "Exact match ratio",
-          "description": "The ratio of instances classified correctly for every label in multilabel task."
+          "description": "The ratio of instances classified correctly for every label in question answering task."
```

Comment:
this is used in multilabel classification too though - so shouldn't we generalize this to just say "The ratio of instances classified correctly for every label"